schemas/json/theme.json

```diff
@@ -1243,6 +1243,13 @@
 				"required": [ "name" ],
 				"additionalProperties": false
 			}
+		},
+		"patterns": {
```

Comment:
Thanks for doing this!

I thought the [reference doc for theme.json](https://developer.wordpress.org/block-editor/reference-guides/theme-json-reference/theme-json-living/) was updated upon the schema changes. Apparently, it doesn't. I've now realized that it only contains data coming from the settings & styles keys, but not from `customTemplates`, `templateParts`, or this new key.

Reply:
https://github.com/WordPress/gutenberg/issues/38620